### DISCUSSION TITLE
mlflow handler implementation

### DISCUSSION THREAD

**Discussion by jpcenteno80** [Project-MONAI/MONAI#2995] [General]
Could you recommend a way to implement a handler to track experiments using mlflow and monai's engines?

Thanks!

**Comment by Nic-Ma** [comment]
Hi @jpcenteno80 ,

Thanks for your interest here.
We didn't use `MLFlow` before, I think you can refer to this handler to implement your own hander:
https://github.com/Project-MONAI/MONAI/blob/0.7.0rc4/monai/handlers/smartcache_handler.py
Just define some callback functions for expected events.

Thanks.

**Comment by wyli** [comment]
Hi @Nic-Ma I think we should build demos with the relevant ignite APIs after v0.7 release
- https://github.com/pytorch/ignite/blob/327cec5544c65d573059a810037f02972328257e/ignite/contrib/handlers/mlflow_logger.py
- https://github.com/pytorch/ignite/blob/9230a7319047b37ce19d956e024fa1b86030c30a/examples/references/classification/imagenet/NOTES_MLflow.md

**Reply by Nic-Ma** [reply]
Hi @wyli ,

Yes, sure. I can implement it later.
Could you please help convert it into a ticket?

Thanjs.
